src/SourceGenerators/Uno.UI.SourceGenerators.netcore.Tests/Verifiers/CSGenerator.cs

```diff
@@ -11,6 +11,13 @@ public static partial class CSharpSourceGeneratorVerifier<TSourceGenerator>
 	{
 		public class Test : CSharpSourceGeneratorTest<TSourceGenerator, MSTestVerifier>
 		{
+			public Test()
+			{
+				TestState.AnalyzerConfigFiles.Add(("/.globalconfig", """
+					is_global = true
+					build_property.Configuration = Release
+					"""));
+			}
```

Comment:
This hack will not be needed when 10932 is merged. Marking this PR as a draft for now.

Reply:
Very nice.